sympy/calculus/util.py

```diff
@@ -10,6 +10,131 @@
 from sympy.functions.elementary.miscellaneous import Min, Max
 
 
+def continuous_in(f, symbol, interval):
+    """
+    Finds the intervals of continuity of a function in a given interval range.
```

Comment:
I would suggest:-> `Return the intervals in the given domain for which the function is continuous.` and then change argument `interval` to `domain` and the function name to `continuous_domain` (`continuous_in(f...)` doesn't read well, at least when read as "`continuous_in` of function for symbol in interval".